### DISCUSSION TITLE
LL HLS on android

### DISCUSSION THREAD

**Discussion by cfeeney5** [bluenviron/gohlslib#195] [General]
Hi,
Im trying to find a video player which can play the low latency HLS on android. I've tried HLS.js but it throws an error (PIPELINE_ERROR_DECODE). It uses exoplayer underneath which says the community implementation isn't supported https://developer.android.com/media/media3/exoplayer/hls. Ive been unable to find any relevant info online. Any help would be appreciated.

**Comment by aler9** [comment]
Hello, `hls.js` is perfectly capable of playing low-latency HLS streams on any Android device. Try to debug the `PIPELINE_ERROR_DECODE` error. The `hls.js` player embedded inside MediaMTX might come handy since it also prints error details on the screen.
